src/editors/mesh.rs

```diff
@@ -193,6 +200,24 @@ fn edit_mesh(
         mesh.objects.remove(i);
     }
 
+    if let Some(i) = mesh_to_duplicate {
+        let mut duplicated_mesh = mesh.objects[mesh_to_duplicate.unwrap()].clone();
```

Comment:
You can just use `i` here instead of unwrapping. This also applies to the other usages and in modl.rs.